src/test/java/com/training/services/tests/GACreateTest.java

```diff
@@ -0,0 +1,68 @@
+package com.training.services.tests;
+
+import com.google.gson.Gson;
+import com.training.base.BaseTest;
+import com.training.services.ga.authenticate.*;
+import com.training.services.ga.create.RequestBodyCreate;
+import com.training.utilities.GetUniqueMailId;
+import com.training.utilities.RestEngine;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import java.util.HashMap;
+import java.util.Map;
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class GACreateTest extends BaseTest {
+    RequestBodyCreate requestBodyCreate;
+    Map<String, Object> headerMap;
+    GetUniqueMailId mail=new GetUniqueMailId();
```

Comment:
lets create a static method for this, so that we can use directly using className instead of creating an object